Source/Core/DolphinQt2/GameList/GameListModel.cpp

```diff
@@ -86,5 +87,7 @@ void GameListModel::RemoveGame(QString path)
 	beginRemoveRows(QModelIndex(), entry, entry);
 	m_entries.remove(path);
 	m_games.removeAt(entry);
+	for (int i = entry; i < m_games.size(); i++)
```

Comment:
I got rid of this bit. I was hoping to avoid O(n) inserts and removals for games, but since they're presented in a list... it wasn't gonna work out. That map was the remnants of my pitiful attempts.